src/components/EditTask/EditTask.jsx

```diff
@@ -33,7 +37,7 @@ const EditTask = ({ task }) => {
           className={styles.Input}
           onChange={handleEdit}
         />
-        <Icon icon="calendar" size={20} />
+        <Datepicker />
```

Comment:
Tarih değişimleri için context tutmamalısın. `editedTask` ın içindeki tarih değerini değiştirmen yeterli.

Reply:
Bunu biraz açabilir misin? Kullandığım contextler olmadan değişimi nasıl yapacağımdan emin olamadım.

Reply:
Datepicker da seçtiğin tarih `editedTask` state tinin içindeki date i değiştirsin.  Bunun için ayrı bir state ya da context kullanmayalım.